README.md

```diff
@@ -41,17 +45,17 @@ To then start using your bot, you can connect your bot to `nxtalkproxyd` like so
     -e NXTALKPROXYD_NXTALKPROXYD_PASSWORD=botpassword \
     -e NXTALKPROXYD_NXTALKPROXYD_RADDR=https://examplenextcloud.com \
     --network nxtalkchatbots \
-    --name nxtalkproxy \
-    -d pojntfx/nxtalkproxy
+    --name nxtalkproxyd \
+    -d pojntfx/nxtalkproxyd
 % docker run \
     -e BOT_JITSI_ADDR=meet.jit.si \
     -e BOT_JITSI_BOT_NAME=botusername \
     -e BOT_JITSI_SLEEP_TIME=20 \
-    -e BOT_NXTALKPROXYD_ADDR=nxtalkproxy:1969 \
+    -e BOT_NXTALKPROXYD_ADDR=nxtalkproxyd:1969 \
```

Comment:
Once these env variables have changed their names, they should probably also be changed over in the actual bot repo.

Reply:
Which is fine for me ...

> This should be `https://pojntfx.github.io/nextcloud-talk-bot-framework/` - simply because the repo is on this path and GitHub doesn't let you configure it.

Reply:
Ok. Will patch it.